pyimzml/compression.py

```diff
@@ -0,0 +1,25 @@
+
+import zlib
+
+class NoCompression(object):
+    name = "no compression"
+    def __init__(self):
+        for name in ('rounding', 'compress', 'decompress'):
+            setattr(self, name, lambda x: x)
```

Comment:
I find it unnecessary obfuscation, given that its easily readable counterpart was already in the codebase.